cpp/ps0003/p0056.cpp

```diff
@@ -4,17 +4,17 @@ using namespace std;
 
 int main()
 {
-   int number ;
-   cout << "enter number \n" ;
-   cin >> number ;
+   // prompting user for number 2/2
+   int number;
+   cout << "enter number \n";
+   cin >> number;
 
-    if (number % 2 == 0)
+   // checking if the number is odd or even 3/3
+   if (number % 2 == 0)
         cout << "Number is even \n";
    else
         cout << "Number is odd \n";
 
-
-
     return 0;
 }
-
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
you can visit the [problem solution](https://onlinegdb.com/PbqxNbSVc) for more

you also can use _ternary operator_:
```cpp
std::cout << "Number is " << ((number % 2 == 0) ? "even" : "odd") << std::endl
```